src/plenoptic/simulate/models/frontend.py

```diff
@@ -339,10 +360,15 @@ def __init__(
     ):
         super().__init__()
         if pretrained:
-            assert kernel_size in [31, (31, 31)], "pretrained model has kernel_size (31, 31)"
+            assert kernel_size in [
```

Comment:
this is ugly. I should probably just rewrite it to use if so it formats nicer...